pkgs/development/python-modules/python-ldap/default.nix

```diff
@@ -2,27 +2,41 @@
 , stdenv
 , buildPythonPackage
 , fetchFromGitHub
-, pyasn1
-, pyasn1-modules
-, pythonAtLeast
 , pythonOlder
-, pytestCheckHook
+
+# build-system
+, setuptools
+
+# native dependencies
```

Comment:
I'd get rid of these comments. All this information is present in the arguments to buildPythonPackage, and if these comments ever get out of date they may do more harm than good.

Reply:
Same comment style as in the python documentation for nixpkgs, coincidentally contributed by me. https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-function-buildpythonpackage-function

To me organizing them like this is preferable to making it an opaque list without context.

Reply:
Oh, look at that. Sorry! Will resolve this :)